dev-clean/benchmarks/klee_fs_lib/FreeBSD.h

```diff
@@ -0,0 +1,457 @@
+#ifndef KLEE_FREEBSD_H
```

Comment:
Does this file have a license?

Reply:
I actually do not know about the "license" concept, I just add this file because it is in klee's POSIX folder.

Reply:
I just see other files have a header like this
```
// This file is distributed under the University of Illinois Open Source
// License. See LICENSE.TXT for details.
```
Maybe let's also include their https://github.com/klee/klee/blob/master/LICENSE.TXT file under this folder

Reply:
Sure.